README.md

```diff
@@ -1,99 +1,82 @@
 
-## How the git workflow changes:
+```bash
+git config --add mergify-cli.stack-trunk origin/branch-name
+```
 
-To uses git-push-stack you have to replace:
+Alternatively, you can set the trunk branch on the fly using the `--trunk` parameter.
 
-```bash
-$ git checkout feature-A
-...do some change and commit...
-$ git push origin feature-A -f
+3. Set up a GitHub OAuth token for Mergify-cli to access the GitHub API. It is recommended to use the `gh` ([gh client](https://cli.github.com/)) installed and authenticated. In this case, Mergify-cli will automatically retrieve the token using `gh auth token`. Alternatively, you can create a [personnal access token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens) with the necessary permissions to create branches and pull requests. Set the token as an environment variable `GITHUB_TOKEN` or provide it on the fly using the `--token` parameter.
 
-$ git checkout feature-B
-$ git rebase feature-A
-$ git push origin feature-B -f
+## Usage
 
-$ git checkout feature-C
-$ git rebase feature-B
-$ git push origin feature-C -f
-```
+To create a stack of pull requests, follow these steps:
+
+1. Create a branch and make your desired changes on that branch.
+2. After making changes on your branch, including any necessary commit message modifications using `git rebase -i`, you can proceed to create your stack of pull requests.
 
-by:
+3. To create the stack, simply run the following command:
 
 ```bash
-...do some change and commit...
-$ git push-stack
+mergify
 ```
 
-## Setup the git commit-msg hook:
+Mergify-cli will automatically handle the creation of individual pull requests for each commit in your feature branch. This allows for a streamlined and efficient process of managing the changes and reviews associated with the stacked pull requests.
 
-```bash
-$ git push-stack -s
-```
+By using Mergify-cli, you no longer need to manually update branches and pull requests when making changes to earlier commits. The tool intelligently tracks and manages the commit index or SHA, ensuring that your stacked pull requests remain up-to-date and synchronized.
 
-## Push Stack your pull request like a pro 🦾
+Remember that one pull request will be created for each commit in your feature branch, facilitating an organized and incremental review process.
 
-```bash
-$ git push-stack
-```
+Feel free to explore additional options and parameters available with the `mergify` command to customize the behavior of Mergify-cli based on your specific needs.
 
-Enjoy!
+That's it! With Mergify-cli, you can automate the creation and management of stacked pull requests, saving time and reducing the chances of errors during the process.
 
-## Installation
+If you have any questions or need further assistance, please refer to the documentation or reach out to the Mergify-cli community for support.
```

Comment:
May be add a link to our community slack ?

Reply:
I don't know. @sileht ? @jd ?

Reply:
Sure thing that sounds great.
It's https://slack.mergify.com